proto/elys/tier/userdata.proto

```diff
@@ -18,4 +18,16 @@ message Pool {
     (gogoproto.customtype) = "cosmossdk.io/math.Int",
     (gogoproto.nullable) = false
   ];
+}
+
+message MembershipTier {
+  string minimum_portfolio = 1 [
+    (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
```

Comment:
if you can add (cosmos_proto.scalar) = "cosmos.Dec", above this line.
You might need to import  "cosmos_proto/cosmos.proto";